internal/command/output_test.go

```diff
@@ -50,6 +50,41 @@ func TestOutput(t *testing.T) {
 	}
 }
 
+func TestOutput_stateStore(t *testing.T) {
+	originalState := states.BuildState(func(s *states.SyncState) {
+		s.SetOutputValue(
+			addrs.OutputValue{Name: "foo"}.Absolute(addrs.RootModuleInstance),
+			cty.StringVal("bar"),
+			false,
+		)
+	})
+
+	statePath := testStateFile(t, originalState)
+
+	view, done := testView(t)
+	c := &OutputCommand{
+		Meta: Meta{
+			testingOverrides: metaOverridesForProvider(testProvider()),
+			View:             view,
+		},
```

Comment:
Is there a reason to avoid `AllowExperimentalFeatures: true` here? 🤔 Just noticed the difference from other commands - no particular other reason behind the question.

Reply:
Ah, I think it was an artefact of me splitting work out into https://github.com/hashicorp/terraform/pull/37894. I'll remove `output` command stuff from this PR

Reply:
Removed now